--- Makefile.am~	Tue Nov 30 01:47:34 1999
+++ Makefile.am	Sun Jun 30 21:31:35 2002
@@ -8,14 +8,7 @@
 COMMON_CFLAGS = -O2 -I$(top_builddir) -I$(top_srcdir) -D_REENTRANT @GTK_CFLAGS@
 COMMON_SRC = blur_scope.c blur_scope.h config.c
 
-if ARCH_X86
-
-EXTRA_CFLAGS = -DX86_ASM_OPT
-EXTRA_SRC = blur_8.s
-
-endif
-
-EXTRA_DIST = blur_8.s xmms_logo.xpm
+EXTRA_DIST = xmms_logo.xpm
 
 CFLAGS = $(COMMON_CFLAGS) $(EXTRA_CFLAGS)
 libbscopemax_la_SOURCES =  $(COMMON_SRC) $(EXTRA_SRC)
